packages/ts/package.json

```diff
@@ -5,11 +5,12 @@
   "main": "lib/src/index.js",
   "types": "lib/src/index.d.ts",
   "scripts": {
-    "test": "./node_modules/ts-mocha/bin/ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts",
-    "test-cs": "./node_modules/ts-mocha/bin/ts-mocha -p ./tsconfig.json -t 1000000 tests/test.canSwap.ts",
-    "test-tbc": "./node_modules/ts-mocha/bin/ts-mocha -p ./tsconfig.json -t 1000000 tests/test.tokenSwap.ts",
-    "test-token": "./node_modules/ts-mocha/bin/ts-mocha -p ./tsconfig.json -t 1000000 tests/test.token.ts",
-    "test-freeze": "./node_modules/ts-mocha/bin/ts-mocha -p ./tsconfig.json -t 1000000 tests/test.freezeTbc.ts",
+    "test": "npx ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts",
```

Comment:
kind of a nit, but given that `test` and `test-all-js-library` do the same thing should we remove `test` as `test-all-js-library` is more expressive?

Reply:
Ohh I can DRY this up just a bit. Doing that now.